incalmo/cli/README.md

```diff
@@ -0,0 +1,41 @@
+# Incalmo CLI
```

Comment:
Why is all this part of the PR? I think this got the branch got a bit messed up.